extensions/amp-list/0.1/amp-list.css

```diff
@@ -22,27 +22,31 @@ amp-list[load-more] [load-more-end].amp-visible {
   width: 100%;
 }
 
-[load-more-button], [load-more-failed], [load-more-loading] {
-  text-align: center;
+amp-list[load-more] [load-more-loading].i-amphtml-default-ui,
+amp-list[load-more] [load-more-failed].i-amphtml-default-ui,
+amp-list[load-more] [load-more-button].i-amphtml-default-ui {
   height: 80px;
-
   padding: 12px 0px;
   box-sizing: border-box;
 }
 
-[load-more-button], [load-more-failed], [load-more-loading],
+/* TODO: Standardize the system font string we use across the code base,
```

Comment:
TODOs in the codebase generally need to have an attached issue and are filed as `TODO (#issueNumber): blablabla`. It would be a good idea to have a tracking issue for this and also get Abby's feedback, since we currently have a few different fonts floating around (e.g. datepicker).

Reply:
Yes, Abby and I chatted about this yesterday - I have a plan to push this forward. I'll file a ticket just as a placeholder and add it to the TODO.